BedrockCommand.cpp

```diff
@@ -253,3 +253,10 @@ void BedrockCommand::postPoll(fd_map& fdm, uint64_t nextActivity, uint64_t maxWa
         transaction->manager.postPoll(fdm, *transaction, nextActivity, maxWaitMS);
     }
 }
+
+void BedrockCommand::setTimeout(uint64_t timeoutDurationMS) {
```

Comment:
We've added this so that we can modify a command's timeout to be no more than 20 seconds when it gets sent to the sync thread, hoping that it will then timeout and the sync thread can continue before it hits 30s and loses quorum.